Source/Morris.Reducible/Reducer.Result.cs

```diff
@@ -5,6 +5,6 @@ public static partial class Reducer
 	public record struct Result<TState>(bool Changed, TState State)
 	{
 		public static implicit operator (bool changed, TState state)(Result<TState> result) => (result.Changed, result.State);
-		public static implicit operator Result<TState>((bool changed, TState value) value) => new Result<TState>(value.changed, value.value);
+		public static implicit operator Result<TState>((bool changed, TState value) value) => new(value.changed, value.value);
```

Comment:
This is okay, but could you also change the parameter names to `TState state` and the param itself to `tuple`

Reply:
Changed